wagtail/query.py

```diff
@@ -179,6 +185,136 @@ def is_specific(self):
             (SpecificIterable, DeferredSpecificIterable),
         )
 
+    def select_related(self, *fields, for_specific_subqueries: bool = False):
+        """
+        Overrides [Django's native `select_related()` implementation](https://docs.djangoproject.com/en/stable/ref/models/querysets/#django.db.models.query.QuerySet.select_related)
```

Comment:
We don't support Markdown in docstrings yet, so this needs to be in rST syntax, e.g.

```suggestion
        Overrides Django's native :meth:`~django.db.models.query.QuerySet.select_related` implementation
```

For inline code, you need to use double backticks instead of single. And for code blocks, you need to use `.. code-block:: python`.

Alternatively, if you want to write the docs in Markdown instead, you can move it to the `.md` file instead of using a docstring.

Reply:
Much appreciated @laymonage. Those CI failures had me really stumped!